R/data_analysis_page.R

```diff
@@ -0,0 +1,2146 @@
+                    fixedRow(
+                      p("Options to change limits of the x-axis:"),
+                      column(
+                        width = 6,
+                        align = 'center',
+                        numericInput(inputId = ns('bayesmin_sub'), label = "Minimum", value = 0.1)
+                      ),
+                      column(
+                        width = 6,
+                        align = 'center',
+                        numericInput(inputId = ns('bayesmax_sub'), label = "Maximum", value = 5)
+                      )
+                    ),
+                    tags$style(
+                      "#ref_change_bay {
+                        background-color: #ffd966;
+                        display:block;
+                      }"
+                    ),
+                    textOutput(outputId = ns("ref_change_bay")),
+                    p("Model fit:"),
+                    tableOutput(outputId = ns("dic_sub")),
+                    textOutput(outputId = ns("text_gemtc_sub")),
+                    br(),
+                    br(),
+                    radioButtons(
+                      inputId = ns('format4'),
+                      label = 'Document format',
+                      choices = c('PDF', 'PNG'),
+                      inline = TRUE
+                    ),
+                    downloadButton(outputId = ns('downloadBaye_plot_sub'))
+                  )
+                )
+              ),
+              tabPanel(
+                title = "3b. Comparison of all treatment pairs",
+                helpText("Please note: if you change the selections on the sidebar, you will need to re-run the primary and/or sensitivity analysis from the 'Forest Plot' page."),
+                p(
+                  tags$strong(
+                    "In contrast to the 'comparison of all treatment pairs' tab in the frequentist NMA results,
+                    this table only contains the estimates from the network meta analysis,
+                    i.e. does not contain estimates from pairwise meta-analysis which only contains direct evidence.
+                    If you would like to obtain the pairwise meta-analysis results, please run 3d. Nodesplit model"
+                  )
+                ),
+                br(),
+                p(tags$strong("Treatment effects for all studies: comparison of all treatment pairs.")),
+                tableOutput(outputId = ns("baye_comparison")),
+                downloadButton(outputId = ns('downloadbaye_comparison')),
+                br(),
+                br(),
+                p(tags$strong("Treatment effects with studies excluded: comparison of all treatment pairs.")),
+                tableOutput(outputId = ns("baye_comparison_sub")),
+                downloadButton(outputId = ns('downloadbaye_comparison_sub'))
+              ),
+              tabPanel(
+                title = "3c. Ranking Panel",
```

Comment:
The relative effects section of the ranking panel is also not displaying (all studies & sensitivity analysis)

Reply:
I have seen that happen, but I've not been able to reproduce it today. I did find that the plot appeared when the window was resized, so it's not a broken output.

Reply:
It's working for me again now ...